web-pages/whats-new.html

```diff
@@ -120,6 +120,7 @@
 
 	   <h1>WATCH THIS VIDEO</h1>
 	   <span>WE DO THINGS IN IT</span>
+     <p>A small change here.</p>

```

Comment:
Not sure about this sentence, I don't like the way it looks.

Reply:
I updated the sentence to sound more professional.